trade-finance-manager-ui/server/controllers/utilisation-reports/add-to-an-existing-payment/get-link-to-premium-payments-tab.test.ts

```diff
@@ -0,0 +1,51 @@
+import { getLinkToPremiumPaymentsTab } from './get-link-to-premium-payments-tab';
+
+console.error = jest.fn();
```

Comment:
What's the reason for stubbing `console.error`?

Reply:
I think this is just to prevent error messages from cluttering the test output

This has been done across most of the services (TFM, GEF, portal, central)